volmdlr/wires.py

```diff
@@ -1550,6 +1550,31 @@ def get_geo_points(self):
             points.update(primitive.get_geo_points())
         return points
 
+    def to_polygon(self, angle_resolution, discretize_line: bool = False):
+        """
+        Transform the contour_mixin to a polygon, COPY/PASTE from Contour2D.
+
+        :param angle_resolution: Number of points per radians.
+        :type angle_resolution: float
+        :param discretize_line: Boolean indicating whether the line segments should be discretized or not.
+        :type discretize_line: bool
+        :return: The discretized version of the contour.
+        :rtype: ClosedPolygon2D
+        """
+
+        polygon_points = []
+
+        for primitive in self.primitives:
+            if isinstance(primitive, volmdlr.edges.LineSegment2D) and not discretize_line:
+                polygon_points.append(primitive.start)
+            else:
+                polygon_points.extend(primitive.discretization_points(angle_resolution=angle_resolution)[:-1])
+
+        if isinstance(self, Contour2D):
+            return ClosedPolygon2D(polygon_points)
+        else:
+            return ClosedPolygon3D(polygon_points)
+
```

Comment:
Also, there exists a method doing the same in ClosedPolygon2D, could you also delete it then? This one will also replace it. For the last return, You can remove the else statement.

Reply:
@sbendjebla  Yes, Sorry